ocs_ci/ocs/utils.py

```diff
@@ -797,6 +797,7 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False, status_failure=Tru
             allows better naming for folders under logs directory
 
     """
+    return
```

Comment:
This is probably a leftover from testing.

Reply:
done